Testing/functional/tests/cypress/integration/ui/errors/tooManyRequestsAlerts.js

```diff
@@ -135,7 +153,18 @@ describe("Public Vaccine Card Downloads", () => {
         cy.intercept("GET", "**/PublicVaccineStatus/pdf", {
             statusCode: 429,
         });
-        cy.enableModules(["Immunization", "PublicVaccineDownloadPdf"]);
+        cy.configureSettings(
+            {
+                covid19: {
+                    publicCovid19: {
+                        enableTestResults: true,
+                        showFederalProofOfVaccination: true,
+                    },
+                },
+            },
+            ["Immunization"]
+        );
+        //cy.enableModules(["Immunization", "PublicVaccineDownloadPdf"]);
```

Comment:
I don't think enableTestResults needs to be set for this test.  Also, the commented out line can be removed.

Reply:
Fixed in AB#14995